__tests__/app.test.js

```diff
@@ -43,4 +42,25 @@ describe("GET", () => {
         });
     });
   });
+
+  describe('"/api/reviews/:review_id: should return specified review from id"', () => {
+    test("200: for returning review from id", () => {

```

Comment:
we want to add test for a 404 when review_id is valid type but does not yet exist i.e. /api/reviews/9999